code/game/machinery/vending.dm

```diff
@@ -103,6 +103,12 @@
 	var/list/restock_blocked_items = list() //Items that can not be restocked if restock_items is enabled
 	var/random_itemcount = 1 //If the number of items should be randomized
 
+	var/cooling = 0 //Whether or not to vend products at the cooling temperature

```

Comment:
Variable name does not match comment. That is to say, the comment makes no sense and the variable looks like it's something which controls if the contents are being cooled/heated.